### ISSUE TITLE
The commands of "List" and "NLST" get wrong arg, and they should can be applied to not only directories but also files

### ISSUE THREAD

**Issue by usernameiooo** [Guichaguri/MinimalFTP#13]
Dear developer, I may find some problems in the program, please consider:<br/>
In your code, you use the function "com.guichaguri.minimalftp.handler.FileHandler.list(String args[]) ,line 230" to handle a receiving LIST command .  But line 236<br/>
    ```
        Object dir = args.length > 0 && !args[0].equals("-l") && !args[0].equals("-a") ? getFile(args[0]) : cwd;
    ```
<br/>may gets wrong dir. 
In my opinion, if args[0] equals "-a", we should use arg[1] to get file rather than cwd. In this case, ftp client may use a Path as arg[1] to represent the directory, or "." to represent cwd. <br/>
And the second problem is that the function response wrong code to client when the path is not directory.   Actually, some ftp client would use LIST command to get information of a file. So, ftp servers so response information in this case. When receiving a LIST command whith a path arg representing a file, just response like that the file is a directory than only has a file (itself) . I have read the source code of    "org.apache.ftpserver.command.impl.LIST", which handles the LIST command in a Apache FTP server, and I find it do so.  <br/>
The similar function nlst(String args[]) that handles NLST command may have same problems.

**Comment by Guichaguri** [comment]
I agree, although that's how the list command works on Linux, the specification does not allow any kind of option for these commands. The specification states that the first parameter is always the path.

This workaround has been implemented like this because clients such as the old Chromium FTP client used to pass these options (and only these options, no paths) against the specs.

But I welcome any improvement to this workaround looking for the first argument that is not an option, since there may be other clients that also pass options :)

**Comment by Guichaguri** [comment]
Implemented in 567d60c5995bb6d5fbb5c1472fc913c7914da0dd
